.codebuild/scripts/lint_pr.sh

```diff
@@ -8,5 +8,5 @@ if [ -z "$PR_NUM" ]; then
 fi
 
 # get PR file list, filter out removed files, filter only JS/TS files, then pass to the linter
-curl -fsSL https://api.github.com/repos/$PROJECT_USERNAME/$REPO_NAME/pulls/$PR_NUM/files | jq -r '.[] | select(.status!="removed") | .filename' | grep -E '\.(js|jsx|ts|tsx)$' || true | xargs yarn eslint
+curl -fsSL https://api.github.com/repos/$PROJECT_USERNAME/$REPO_NAME/pulls/$PR_NUM/files | jq -r '.[] | select(.status!="removed") | .filename' | (grep -E '\.(js|jsx|ts|tsx)$' || true) | xargs yarn eslint --quiet
```

Comment:
The parenthesis is required here. Otherwise it won't emit the lint error. Also the `--quiet` is added to silent the warnings, which is the same as CCI